nlweb-template/public/fp-chat-interface.js

```diff
@@ -809,6 +809,19 @@ class ModernChatInterface {
     }
   }
 
+  markdownToHtml(markdown) {
```

Comment:
Markdown is a bit of a beast, and I'm pretty sure this won't actually capture the full spec, but I'm assuming this also isn't actually your file, so happy to leave it as it is if you are happy!